### DISCUSSION TITLE
Can't use XBox Controller

### DISCUSSION THREAD

**Discussion by The-Michael-Chen** [isaac-sim/IsaacLab#1590] [General]
I tried to use a XBox controller to control a robot in IsaacLab, but the controller signals were not being read. 
I tested the controller on hardwaretester.com and it is connected and sending signals properly. 
I'm trying to use this script: https://docs.robotsfan.com/isaaclab/_modules/omni/isaac/lab/devices/gamepad/se3_gamepad.html
More specifically, I am running the script from this repo https://github.com/YonduAI/Galaxea_Lab/blob/master/source/standalone/galaxea/teleop/teleop_biman_sync.py
I don't get any errors and the keyboard is working fine. I also tried another off the shelf controller and it didn't work either. When I printed out the raw values the controller was getting I got all zeros even while moving the joysticks. 
The command I ran was: `./isaaclab.sh -p source/standalone/galaxea/teleop/teleop_biman_sync.py --task Isaac-Lift-Bin-R1-IK-Rel-v0`

**Comment by shanereetz** [comment]
The teleop script above expects a `--device` flag to use the gamepad rather than the default input device of a keyboard, see [line 111 ](https://github.com/YonduAI/Galaxea_Lab/blob/master/source/standalone/galaxea/teleop/teleop_biman_sync.py#L111)

Can you try again with this command?
`./isaaclab.sh -p source/standalone/galaxea/teleop/teleop_biman_sync.py --device gamepad --task Isaac-Lift-Bin-R1-IK-Rel-v0`

**Comment by RandomOakForest** [comment]
Thank you for posting this. Let us know if the suggestion above worked for you, or if you need further help.

**Comment by PinkPanther-ny** [comment]
Nope, it's not working. Me using Se2Gamepad always receive [0. 0. 0.] when doing print(self.gamepad.advance())   (self is a class inherit ManagerBasedRLEnv). Also, my gamepad is recognized on this tester site: [https://hardwaretester.com/gamepad](url)
